cfbs/commands.py

```diff
@@ -513,6 +519,38 @@ def build_step(module, step, max_length):
             touch(dst)
         assert os.path.isfile(dst)
         sh(f"cat '{src}' >> '{dst}'")
+    elif operation == "directory":
+        src, dst = args
+        if dst in [".", "./"]:
+            dst = ""
+        print(f"{prefix} directory '{src}' 'masterfiles/{dst}'")
```

Comment:
`.format()`

Reply:
ha! yes! that's a copy/paste is why it got through ;)